examples/Cargo.toml

```
@@ -26,6 +27,10 @@ path = "middleware_ws.rs"
 name = "middleware_http"
 path = "middleware_http.rs"
 
+[[example]]
+name = "multi-middleware"
+path = "multi-middleware.rs"
```

All other examples use underscores

```suggestion
name = "multi_middleware"
path = "multi_middleware.rs"
```